qiskit/transpiler/passes/scheduling/scheduling/base_scheduler.py

```diff
@@ -70,8 +70,9 @@ def _get_node_duration(
             duration = dag.calibrations[node.op.name][cal_key].duration
 
             # Note that node duration is updated (but this is analysis pass)
-            node.op = node.op.to_mutable()
-            node.op.duration = duration
+            op = node.op.to_mutable()
+            op.duration = duration
+            node.op = op
```

Comment:
This worries me that the IBM Provider scheduling passes will be doing similar things and this PR will break them.

Reply:
It's only needed if if you set the build time flag. I will update the upgrade the release note to indicate this kind of compatibility thing is only in that case and say it will change the default in 2.0.